atlas_densities/app/mtype_densities.py

```diff
@@ -217,13 +217,21 @@ def _check_config_sanity(config: dict) -> None:
     required=True,
     help="Path to output directory. It will be created if it doesn't exist already.",
 )
+@click.option(
+    "--joblib-n-jobs",
```

Comment:
good call; can we change the name to ```--n-jobs```, so that we can switch out `joblib` in the future if we want?